app/appshell/app.js

```diff
@@ -210,8 +211,8 @@ app.getRemoteDebuggingPort = function () {
 };
 
 app.getUserDocumentsDirectory = function () {
-    // TODO: implement
-    throw new Error("app.getUserDocumentsDirectory not implemented");
+    var _path = path.resolve(electronApp.getPath("home"), "Documents");
```

Comment:
Well guys, I've implemented this only because it was in the tests; but now by an easy search I found out that Brackets never uses it :) so i'll remove both implementation and the test ...